### ISSUE TITLE
Question about how to use the layer that does not exist in op_handler.

### ISSUE THREAD

**Issue by ragAgar** [shap/shap#1344]
Hi,

Thank you for sharing the great implementation.

I have a question about how to use the Pytorch layer that does not exist in `op_handler` in `deep_pytorch.py`.

For example, can I use PyTorch LayerNorm layer instead of BatchNorm1d by adding `op_handler["LayerNorm"] = linear1_d` to the last line of `deep_pytorch.py` ?

Thank you!
